pages/1.12/cli/autocompletion/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: layout.pug
+navigationTitle:  CLI Autocompletion
+title: CLI Autocompletion
+menuWeight: 7
+excerpt: Enabling autocompletion for the CLI
+enterprise: false
+---
+
+Autocompletion was realeased with the 1.12 release so the CLI must be of version 0.7.x or greater. \
+You will also need to have the `dcos` executable in your `$PATH`.\
+
+Autocompletion for the CLI will only complete commands that are available at the time of completion. So if you are not attached to any cluster, only the following commands will be able to complete: `auth`, `cluster`, `config`, `help` and `plugin`. To complete other commands you will have to do be attached to a cluster and added the respective plugins.\
+Completions are available for the CORE CLI plugin and the EE CLI plugin.
+
+# Bash #
+
+
+For the completions to work in `bash` you will need to install `bash-completion`.\
+
+macOS:  `brew install bash-completion` \
+Debian/Ubuntu: `apt-get install bash-completion`.
+
+Now to activate the completions you need to add two lines to your `.profile`:
+```
+[ -f /usr/local/etc/bash_completion ] && . /usr/local/etc/bash_completion
+eval "$(dcos completion bash)"
+```
+
+
+# Zsh #
+
+
+For the completions to work in `zsh` you will need to add the following two lines to your `.zshrc`:
```

Comment:
Replace by: "If you use zsh and want to use the DC/OS CLI autocompletion, you will need to add the following two lines to your `.zshrc`:"